simple/rdm_shared_av.c

```diff
@@ -44,6 +44,7 @@
 static int init_fabric(void)
 {
 	int ret;
+	int other;
```

Comment:
sure, i can rename to `ret2`. i initially called it other because it represents the return value of the other end of the socket pair.